opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -392,6 +395,7 @@ def start_span(
         attributes: typing.Optional[types.Attributes] = None,
         links: typing.Sequence[Link] = (),
         start_time: typing.Optional[int] = None,
+        auto_update_status: bool = True,
```

Comment:
Actually, I'd like to have a more descriptive name for this, but the best I can come up with is `set_error_status_on_exception` which is a mouthful.

Reply:
Ok, fixed!